### DISCUSSION TITLE
Image Resize not works

### DISCUSSION THREAD

**Discussion by zaib347** [dlemstra/Magick.NET#821] [General]
i am ressizing 7 images with exact height width . first three images resize with reverse height and width and rest images correctly resized.
  MagickGeometry size = new MagickGeometry(width, height);
  size.IgnoreAspectRatio = true;
  image.Resize(size);

**Comment by dlemstra** [comment]
I don't have enough information to give an answer. Can you provide an image for the situation where the image is resized correct and one where the size is incorrect? And what are the values of `width` and `height`?

**Reply by zaib347** [reply]
i solved my problem , there is issue in image orientation thats why height and width values replace  each other
